sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonReportSensor.java

```diff
@@ -38,28 +36,29 @@
   protected static final Logger LOG = LoggerFactory.getLogger(PythonReportSensor.class);
 
   protected Settings conf = null;
-  protected ModuleFileSystem fileSystem;
+  protected FileSystem fileSystem;
 
-  public PythonReportSensor(Settings conf, ModuleFileSystem fileSystem) {
+  public PythonReportSensor(Settings conf, FileSystem fileSystem) {
     this.conf = conf;
     this.fileSystem = fileSystem;
   }
 
   public boolean shouldExecuteOnProject(Project project) {
-    return !fileSystem.files(FileQuery.onSource().onLanguage(Python.KEY)).isEmpty();
+    return fileSystem.hasFiles(fileSystem.predicates().hasLanguage(Python.KEY));
```

Comment:
The new code is not equivalent to the old one: FileQuery.onSource() should be transformed in a predicate on file type.